server/models.js

```diff
@@ -0,0 +1,31 @@
+const db = require('../database/connect');
+
+module.exports = {
+    listings: {
+        get: function(params, callback) {
+            Reviews.find({listing_id: params}).exec(function(err, result) {
+                callback(err, result);
+            })
+        }
+    },
+    reviews: {
+        get: function(params, callback) {
+            Reviews.find({$and: [
+                {listing_id: params[0]},
+                {"body": {$regex: new RegExp(params[1], "i")}}
+            ]}).exec(function(err, result) {
+                callback(err, result);
+            })
+        }
+    }
+}
+const getAllProducts = (callback) => {
```

Comment:
This is being defined after your module.exports, are you planning on using it anywhere else?

Reply:
@matt-leyrat thanks for pointing this out, decided that it's not worth to do model-controller-router in such small app, deleted these files. My index.js inside server folder takes care of everything!

Reply:
Ok, you might end up separating everything later but this works for now.